integrations/optimizely/HISTORY.md

```diff
@@ -1,3 +1,8 @@
+3.6.0 / 2020-06-04
+==================
+
+  * Exposed `window.optimizelyEdge` so customers with Performance Edge are able to track Performance Edge related experiments in Segment.
```

Comment:
```suggestion
  * Track conversion events using `window.optimizelyEdge` instead of `window.optimizely` if Optimizely Performance Edge is running on the page.
```